setup.py

```diff
@@ -16,8 +16,8 @@
     description='An API queueing system built at Plivo.',
     long_description=open('README.md').read(),
     install_requires=[
-        'redis==2.10.1',
-        'msgpack-python==0.4.2'
```

Comment:
we also need to update msgpack-python version in services (plivo-api-voice & plivo-api-sms), which uses sharq-client.